packages/cursor/init-gpg.sh

```
@@ -0,0 +1,110 @@
+EOF
+
+gpgconf --kill gpg-agent 2>/dev/null || true
+gpgconf --launch gpg-agent
+
+echo "$GPG_PRIVATE_KEY_BASE64" | base64 -d | gpg --batch --import
+
+# Extract identity from secrets or from the key UID.
+KEY_UID=$(gpg --with-colons --list-secret-keys 2>/dev/null | awk -F: '/^uid:/ {print $10; exit}')
+
+if [[ -n "${MY_GIT_EMAIL:-}" ]]; then
+    GIT_USER_EMAIL="$MY_GIT_EMAIL"
```

Shellcheck (0.11.0)

[style] Prefer putting braces around variable references even when not strictly required.

(SC2250)